### ISSUE TITLE
Building Features should have a strict location in menu

### ISSUE THREAD

**Issue by disarticulate** [Anuken/Mindustry-Suggestions#4836]
### Describe the content or mechanics you are proposing.

![image](https://github.com/Anuken/Mindustry-Suggestions/assets/17464125/e423b149-6261-4423-8838-d16de368b3fc)

Doing the campaign and building the tech tree, factories above, should have a fixed location when the menu is open. EG, if it's not researched yet, it should just be empty or [?]. It should never change location.

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

I find that it's confusing that unlocking features/buildables show up in "different" locations in these menus, so every couple of unlocks my physical memory, eg, where I move me mouse, suddenly changes. I'm certain a lot of people starting the game are just using the mouse to get things. While I know there's numberings and all that, I'm not trying to Tetris my way through these games. But even having built hundred smelters, it's confusing now that there's a multi-press where it used to just be a silicon smelter and visually there's no indicate.

tldr: buildable menu should fix locations for everything and just show the full menu. You could add all the [?] tiles or hints if you feel it fleshes it out, but the major issue is my muscle/location memory is quicker than anything else when I'm mousing around.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by egfthomas** [comment]
Agree!
